### PR TITLE
docs: Update main README file

### PR DESCRIPTION
**Pull request by herrardo**
EX-3970

# Pull Request Template #

Update main README file

## Motivation and Context ##

To have a better README file. As the contributing files, this is a work in continuous progress, and i would appreciate any suggestion to improve our README thinking on this as an open-source project.

## Type of change ##

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [X] This change requires a documentation update

## What is the destination branch of this PR? ##

main

## How Has This Been Tested? ##

Check typos

## Checklist: ##

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [X] I have made corresponding changes to the documentation
- [X] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,67 @@
...
+* [@empathyco/x-components](https://github.com/empathyco/x/tree/main/packages/components) - This is the core package of this project. 
+These are standalone and configurable building blocks that allow you to quickly construct the search UI for your shop. 
+You can create a smooth, personalised search and discovery experience, while significantly minimising development time. (To be moved here soon) 
+* [@empathyco/x-archetype](https://github.com/empathyco/x/tree/main/packages/archetype) - This package is the Empathy vision of the 
+perfect X-Components mixing. A project showing the power of the components ready to connect to any Search API with customizable layout through Design Tokens. 
+This is a perfect example to know how to use the X-Components. (To be moved here soon) 
+
+## Roadmap
+
+2021 will be the year to consolidate Interface X, the roadmap includes this highlighted milestone:
```
Comment (pmareke) [inline README.md:30]: `these highlighted milestones`

File: README.md [hunk 2]
```diff
@@ -1,6 +1,67 @@
...
+* URL Management to set X-Components state based on URL parameters
+* Create a tagging system to track user interactions
+* Search box power ups as animated suggestions or forbidden character set to prevent code injection
+* Publish a extensive documentation covering functional and technical sides.
+* To be an open source project! 
+
+## Fast, flexible, and straightforward 
+
+* **Easy-to-add interface layer**. Installation and setup of the interface is simple: just add a few lines of code.
+* **Use it anywhere.** Search interface layer that is easy to integrate into any website, with X Components ready to use in your Vue.js and React projects.
```
Comment (pmareke) [inline README.md:42]: ` **Use it anywhere.** Search interface layer is easy to integrate into any website...` I'm not sure about this one but the `that` word sounds weird.

**Review by pmareke — COMMENTED** [review]
Nice job crack.

File: README.md (hunk 2 above)
Comment (herrardo) [reply README.md:42]: It would have more sense saying:

`A search interface layer that is...`

File: README.md (hunk 1 above)
Comment (herrardo) [reply README.md:30]: Done

**Review by pmareke — APPROVED** [review]

**Review by tiborux — APPROVED** [review]

**Pull request merged**